nova/virt/vmwareapi/volumeops.py

```diff
@@ -27,12 +27,25 @@
 from nova.i18n import _
 from nova.virt import driver
 from nova.virt.vmwareapi import constants
+from nova.virt.vmwareapi.session import StableMoRefProxy
 from nova.virt.vmwareapi import vm_util
 
 CONF = nova.conf.CONF
 LOG = logging.getLogger(__name__)
 
 
+class VolumeMoRefProxy(StableMoRefProxy):
+    def __init__(self, connection_info_data):
+        volume_ref_name = connection_info_data['volume']
```

Comment:
Since we get this from a moref by calling `get_moref_value()`, I'd rather name it `volume_ref_value`.

Reply:
Done